app/components/repo-actions.js

```diff
@@ -66,9 +66,10 @@ export default Component.extend({
   isHighPriority: or('item.priority', 'item.build.priority'),
   isNotAlreadyHighPriority: not('isHighPriority'),
   hasPrioritizePermission: or('item.permissions.prioritize', 'item.build.permissions.prioritize'),
-  canPrioritize: and('item.notStarted', 'isNotAlreadyHighPriority', 'hasPrioritizePermission'),
   insufficientPermissions: not('userHasPushPermissionForRepo'),
   tooltips: or('labelless', 'mobilelabels'),
+  noSmallIcons: not('tooltips'),
```

Comment:
This is not quite it. As far as I understand, we still wan't to have this button on build lists, so it should not appear only for job. If you'd like to distinguish between full and icon display, please use only `labelless` property. Although, I'd suggest slightly different approach here. As there's a specific place where we'd like to hide these icons, independently of the state, it would be much more reliable to just add another attribute to the component, call it something like `showPriorityButton: true` (true by default, because by default we want the buttons to show), and then just set this property to false only where we want them to be hidden - particularly, in jobs list, e.g.
```hbs
<RepoActions
  @job=this.job
  @labelles=true
  @showPriority=false
/>
```
And then in template you can show the button when `{{#if and(this.canPrioritize this.showPriority)}}` - and that's it :)

How does it sound to you?

Reply:
Hello, @artursmirnov I get it, I have made the changes accordingly and verified the working as well. May you kindly check the same.